### ISSUE TITLE
OAuth 2.0 Scope may need expansion

### ISSUE THREAD

**Issue by duganj** [loghound/Fitbit-for-Google-App-Script#22]
As of 2016-02-22 I was getting failures of the script running successfully. I revoked authorization and forced re-authorization with a change in scope to include:

.setScope('activity nutrition sleep weight profile settings')

Some of the above may be unnecessary, but I was casting a broad net. After the re-authorization I was able to run the script again.

See:
https://dev.fitbit.com/docs/oauth2/#scope

**Comment by MonoBuoy** [comment]
Same issue here. Problems with early OAuth 2.0 implementation?

**Comment by meredithkfwilliams** [comment]
Yep, definitely issue with early OAuth implementation. This resolved issue for me too.

**Comment by meredithkfwilliams** [comment]
Check out: https://github.com/meredithkfwilliams/GoogleSheetsFitBitData
